test-requirements.txt

```diff
@@ -1,4 +1,5 @@
-flake8==6.0.0
 bandit==1.7.4
```

Comment:
@ccrisan - don't we need `bandit[toml]` here since we're using `pyproject.toml` file for bandit config?

Reply:
Apparently not :) I pulled `bandit` and `toml` and it worked perfectly.